README.md

```diff
@@ -1,15 +1,34 @@
 # Lokalise - the unofficial node lokali.se client.
 
-This is all still very WIP.
-
-## How to use
+### Configuration
```

Comment:
I am a huge stickler for semantics, sometimes even at the cost of aesthetics. Let's leave this as a second level heading?

Reply:
👍